source/package.json

```diff
@@ -0,0 +1,115 @@
+    "deploy:client": "yarn export && yarn update-client-stack-variables && AWS_REGION=$npm_package_stack_region STACKNAME=$npm_package_stack_stackname cdk deploy --toolkit-stack-name DocumentUnderstandingCDKToolkit --require-approval never -v true -a \"node bin/deploy-client-stack.js\"",
+    "deploy:setup-user": "AWS_REGION=$npm_package_stack_region node bin/user-setup.js",
+    "deploy:setup-samples": "AWS_REGION=$npm_package_stack_region bash ./bin/upload-samples.sh",
+    "deploy:show": "AWS_REGION=$npm_package_stack_region node ./bin/client-stack-url.js",
+    "deploy-all": "yarn compile-ts-stacks && yarn bootstrap && yarn deploy:backend && yarn deploy:client && yarn deploy:setup-samples",
+    "deploy": "yarn deploy-all && yarn update-pdf-lambda-code",
+    "deploy:frontend": "yarn deploy:client && yarn deploy:setup-samples && yarn deploy:show",
+    "destroy": "STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region cdk destroy -v true -a \"node bin/deploy-client-stack.js\" && STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region USER_EMAIL=$npm_package_email cdk destroy -v true",
+    "update-client-stack-variables": "bash ../deployment/custom-deployment/bin/stack-variables-to-client.sh",
+    "update-pdf-lambda-code": "bash ../deployment/custom-deployment/bin/update-pdf-lambda-code.sh",
+    "dev": "next app",
+    "build": "STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region yarn pre-build && next build app",
+    "export": "yarn build && next export app",
+    "pre-build": "STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region node ./bin/pre-build.js",
+    "license-report": "legally --plain > license-report.txt && echo \"$(tail -n +4 license-report.txt)\" > license-report.txt",
+    "prettier": "prettier --write '**/*.{js,md,yaml}'",
+    "lint": "yarn lint:js && yarn lint:css",
+    "lint:css": "sass-lint --verbose --no-exit",
+    "lint:js": "eslint './**/*.js' --max-warnings 0"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-apigateway": "^1.14.0",
+    "@aws-cdk/aws-cloudformation": "^1.14.0",
+    "@aws-cdk/aws-cognito": "^1.14.0",
+    "@aws-cdk/aws-dynamodb": "^1.14.0",
+    "@aws-cdk/aws-elasticsearch": "^1.14.0",
+    "@aws-cdk/aws-events": "^1.14.0",
+    "@aws-cdk/aws-iam": "^1.14.0",
+    "@aws-cdk/aws-lambda": "^1.14.0",
+    "@aws-cdk/aws-lambda-event-sources": "^1.14.0",
+    "@aws-cdk/aws-s3": "^1.14.0",
+    "@aws-cdk/aws-s3-deployment": "^1.14.0",
+    "@aws-cdk/aws-sns": "^1.14.0",
+    "@aws-cdk/aws-sqs": "^1.14.0",
+    "@aws-cdk/core": "^1.14.0",
+    "@types/node": "^12.12.5",
+    "@zeit/next-sass": "^1.0.1",
+    "aws-amplify": "^1.1.28",
+    "aws-cdk": "^1.14.0",
+    "classnames": "^2.2.6",
+    "date-fns": "^1.30.1",
+    "dotenv": "^8.0.0",
+    "express": "^4.17.1",
+    "inquirer": "^6.4.1",
+    "lodash": "^4.17.11",
+    "next": "^8.1.0",
```

Comment:
We might have to upgrade to 9.2.2 since the prior versions depend on and older version of serialize-javascript that has a vulnerability
https://www.npmjs.com/advisories/1426

Reply:
we can test out updating this version and make sure it does not break anything. Worth doing separately from this PR

Reply:
Pushing this change to Github without the update will probably open another vulnerability ticket similar to this [one](https://github.com/awslabs/amazon-textract-document-understanding-solution/network/alert/yarn.lock/serialize-javascript/closed)